### DISCUSSION TITLE
Structure of the guide & claims

### DISCUSSION THREAD

**Discussion by olarubaj** [rufuspollock/awesome-crypto-critique#144] [Ideas]
## User story

The main reason I'm visiting the site is that I'd like to get a better understanding of what exactly is web3, crypto, 💩coins and NFTs that everyone is talking about and how important these subjects are for me (an ordinary person). And I don't really know where to start 😧 and the subject seems a bit scary 🫣.

As a user, when entering the guide I'd like to be somewhat led by the hand by the site and embark on an exciting trip around the web3!

**Key questions I'm trying to find answers to are (as an imaginary user, for example):**

1. Web3? Was there even Web2? How is it any different?
2. What exactly is a cryptocurrency and why I should even bother? Why it's even called "crypto"?
3. How does it even work? ELI5?
4. Will crypto ever replace fiat money? Should I sell my TV and run buy some crypto to... right.. to where exactly?
... and many others

The site looks great and seems like there is a lot of quality content in there but...

**Main obstacles a user encounters:**

1. Where should I start? There are so many concepts, so many claims discussed and so many videos, and the library! 🤯
2. How can I find videos, podcasts, articles, recent news, and other curated sources related to specific topics like for example DAOs?
3. ...

## Ideas / possible solutions

1. Moving basic/general concepts (noncontextual) to a separate page - e.g. called the glossary - where the user can look up specific definitions/terms. Maybe it would be better if these are not the first thing that users encounter when visiting the guide but rather if they serve as helpers for understanding discussions around specific claims, articles, podcasts, and videos (notes). Ideally, these basic concepts (definitions) could only show up in a box/window on hovering over an underlined word (clickable as well to go directly to the definition page to learn more).
2. Making claims a part/section of the guide (along with e.g. tutorials section)? The questions asked there are really important ones and IMHO are THE thing that can help make sense of web3 and crypto.
3. Creating another page in place of the claims - the glossary - that would contain all raw definitions and non-contextual concepts?
6. ...

**Comment by SalBayat** [comment]
Some of your questions are not easy to answer concisely and are tied to eachother, so this will be a bit long, but hopefully somewhat helpful. 

### Key Questions:

Web1 and Web2 are different beasts.  Think of Web1 as the original web with a lot of static content.  Web2 was where we started to get more user generated content, ease of use was emphasized and we had the growth of platforms, such as social media or video sites.  Its the traditional web as you know it today.  If you want to learn more about what web2 is, there are lots of resources online that will explain it better than me.  Here is a post from 2005 for example:

https://www.oreilly.com/pub/a/web2/archive/what-is-web-20.html

The definition for web3 depends on who you talk to.  Web3 technology is built on something called blockchain, which is a very specific data structure.  

Think of blockchain as a shared excel spreadsheet where you can only add rows to the end of the file.  You can append data, no deleting.  People who add data to the spreadsheet are rewarded, but to have the privilege of adding entries there are qualifying criteria that prove you have "skin in the game".  Either you have to spend money on electricity and computer hardware to conduct complicated calculations which demonstrate that you've done some work (wasted electricity), or you have to own some tokens, which you've acquired because you paid real money for them, or you were involved in the creation of the blockchain in question which allowed you to give yourself or your friends the tokens.

Let's focus on the largest public blockchain, Bitcoin, which uses the calculation method known as proof-of-work.  Bitcoin isn't typically used for Web3 stuff, as developers prefer more modern blockchains (for reasons we won't go into), however, it is a good place to start if you want to understand some of the core concepts surrounding blockchain.  

Participants in the network are asked to solve a complicated math problem in order to be able to add data to the Bitcoin blockchain.  If you can solve the problem before anyone else, you can append data to the blockchain and, as part of this process, are rewarded with some tokens (known as bitcoin, with a small b).  These tokens are referred to as cryptocurrency.  Participants who conduct proof-of-work calculations are known as Miners, and the process of performing these calculations in the hopes of a reward is known as mining.  The mining rewards are contingent on a Miner adding data to the blockchain, this data consists of transactions by users who are sending bitcoin to one another.  Miners broadcast their solution to the other Miners on the network.  Once a Miner validates a solution, they will ignore any further solutions which are broadcast to them, and will append the data from the successful Miner to their own copy of the blockchain.  It is by this way that consensus is reached on which data should be written to the blockchain (a.k.a. ledger).

Miners are incentivized to do the work appending data as they are paid with bitcoin (tokens/crypto/cryptocurrency).  Who pays them?  The Bitcoin protocol does.  The protocol defines the rules which govern participation in the Bitcoin network, including exactly how the proof-of-work system is coordinated between Miners to figure out who gets to add data and be rewarded.  Once mined, the token rewards are assigned to a unique address owned by the Miner, this address is known as a wallet and it is cryptographically secured so that only the Miner in question has access to it.  Anyone may have a wallet which is secured to them and participate in the sending and receiving of bitcoins.  Please note that all of the bitcoin in existence (all 19million tokens) at one point were mined by a Miner.  If you didn't mine the bitcoin yourself then you either need to pay someone for them, or have them transfer them to you out of the kindness of their heart.  In summary, different entities participate in a public system, using a protocol (rules about how to communicate), in order to coordinate adding data to this excel spreadsheet.  The main use of this data is to track who owns how many bitcoins.  As digital data can be easily copied, the blockchain database ensures that people can't send more tokens than they are in possession of, this is the famous "double spend" problem that Bitcoin solves.  

Bitcoin enables me to send a token to someone without anyone being able to stop that transaction from being processed by the Miners who add the record of the transaction to the blockchain.  This is true as long as no one controls more than 51% of the mining work being conducted, beyond that threshold, a single coordinated group could take control of the network.

Returning to the topic of web3, some argue that by building on top of these blockchain systems, they can enable a web which is decentralized, in that no central party can get in the middle of a transaction.  That is to say that the control and even functionality of existing centralized platforms, like facebook, can be spread out so a single entity isn't responsible for the control and operation of the platform.  This is preferable as it removes intermediaries involved in human interactions.  By removing these intermediaries, interactions, or transactions, can be more efficient and free from potential interference due to the intermediary being corrupt.  

I think it is safe to say that those who are participating in this project do not share the viewpoint above.  We would argue that the blockchain technology that web3 is built on is unsound not only technologically but ethically as well.  Web3 is mostly being used as a buzzword to scam the general public and extract value from them all the while providing nothing innovative or of economic utility.  Blockchain is inefficient, not particularly useful, and is far less democratic than even our deeply flawed corporate system that control many of the platforms we use today (such as github).  

However, blockchain does excel (no pun intended) in several areas.  Blockchain incentive structures make them extremely useful in the operation of ponzi schemes, and engaging in financial fraud of all sorts.  Moreover, because you can participate in these systems with some degree of anonymity, it is useful as a way to engage in criminal activity, as it facilitates ransomware attacks, the drug trade, human trafficking, and tax evasion.  Remember those complex calculations I mentioned above that let you add data to the blockchain?  Well it turns out that they use an enormous amount of electricity, and cause a stupendous amount of e-waste.  So Bitcoin and Ethereum, or what we call Public Permissionless Blockchains, are also really good at environmental devastation.  Bitcoin for example uses as much electricity as Belgium, maybe more, and that's for a total of around 10 transactions per second on a good day.  To put that in perspective, assuming your laptop has an SSD, it could probably handle a few hundred transactions per second.  So regular technologies, systems, and platforms are better than Blockchain for payments, serving web pages, or anything else.  Blockchain is only BETTER when it comes to bad things, what economists would call negative externalities.   

This is the point in the discussion where someone might say "Yes, this is all true, BUT, it's all worth it.  Everything you mentioned is worth it because these blockchain databases, and the web3 applications built on top of them, are **decentralized!**"  

The problem is that not only are most (probably all) of these blockchains not decentralized in terms of how they function technically, but the ownership and control of them is extremely centralized as well.  Cryptocurrency uses technology to create a smokescreen that hides centralized control. The majority of tokens are owned by an extremely small percentage of people, it is even more unequal that our existing economic system.  If you think the richest 1% owning the majority of global wealth is bad, you ain't seen nothing yet.  I don't want to get too deep into the technical weeds here, but if you look at how these systems actually work, how they are controlled through their incentive structure, how their "governance" (control of the system) operates, you find that a very small group of insiders and early adopters control most of the "wealth" inside these systems and the decisions that determine how they operate.  This means that any systems predicated on them, like web3, will also share this fundamental power imbalance.

Moreover, some things are more important than money.  We don't condone slavery, we don't allow people to sell their organs, we don't sell children to the highest bidder at auction, the fire department doesn't charge you for saving you from a burning building.  In most democracies, there aren't à la carte price lists for human rights (at least officially).  At some point in our cultural and sociological evolution we decided that all humans have inherent rights, and it has been a tortuously slow, faltering, and hard won process.  This is an important distinction to crypto systems.  Because of their mining incentive structure, and unequal distribution of resources (wealth), ultimately crypto systems can only serve to increase the value of their tokens.  

The **primary** interest that can be served inside of a cryptocurrency (public permissionless blockchain) system is that of increasing the value of its token.  

Nothing else matters, any interest that does not serve this primary goal will not be represented.  Not human rights, not the equitable distribution of resources, not efficient payments, or protections for users.  Nothing will be, or can, be done except what is in the interest of those who are in control of these systems.  What do those in control of these systems care about?  Money and power.  How do they get more of it?  By increasing the value of the token.  How do they do that?  By having more people participate in, and transfer money into, these fundamentally tyrannical systems that don't actually do anything useful.  

Bitcoin, and blockchain, are designed this way on purpose, with intent to benefit the few at the expense of the many.  This makes them evil.  The people who wish to divide us, whittle away our democracies, and replace our economic, political, and other social systems with this technology are dangerous and must be stopped.   

So do I think Crypto will replace fiat and we will be forced to accept our new masters?  I hope not.  

Monetary policy is the most political thing there is, it's how we decide who gets what.  Ceding control of that power to anonymous, unelected Lords who only care about themselves will destroy democracy and transform it into a feudal tyranny.  The highly unequal economic system we have today is imperfect, but crypto is far worse.  It is ethically regressive.


### Main obstacles I encounter:

Start with this documentary by Dan Olson called Line Goes Up - https://www.youtube.com/watch?v=YQ_xWvX1n9g

Check out https://web3isgoinggreat.com/ by Molly White, and read her blog https://blog.mollywhite.net/blockchain/.  Specifically, read https://www.mollywhite.net/annotations/latecomers-guide-to-crypto and at the bottom of the article you will find a list if contributors, I would suggest following all of them on Twitter.

 Stephen Diel has amazing content on his blog as well - https://www.stephendiehl.com/blog.html

Checkout the interviews relating to crypto on the lifeitself youtube channel, https://www.youtube.com/channel/UCmBq3VO9w_MLTKD_CHsYN-Q, the interview with Cory Doctorow is probably the most relevant.  

The main github page contains some useful links, including ones on DAOs https://github.com/life-itself/web3.  Moreover, check out https://web3.lifeitself.us/, the data contained in this repo serves as the source for information on that website.  Browsing the content through github is more useful if you wish to contribute, from an ease of consumption standpoint it may be easier to browse web3.lifeitself.us but this probably depends on personal preference. 

You can check out the subreddits /r/buttcoin, /r/cryptoreality, or /r/cryptocritical (full disclosure: this is a new subreddit I have just started and its early days).  


### Ideas / possible solutions

I don't think I can comment too much on these, except to ask if you have browsed https://web3.lifeitself.us?  Everything is linked on the website, including definition of terms, so I feel like your questions make sense if you have only seen the github repo?  

Maybe you could tell us how you made your way here?  Google search? Twitter?

**Reply by rufuspollock** [reply]
Great points @SalBayat! Thanks for sharing.

**Comment by theo-cox** [comment]
@olarubaj agree on usability. My sense would be shifting [some of the?] guide to glossary, prioritizing making the notes pages visible and navigable (as this is where more detailed explanation is). @EilidhRoss1 and I are also working on creating answers to FAQs (https://github.com/life-itself/web3/issues/143) which we can feature, which will address these questions more directly

**Comment by theo-cox** [comment]
These issues/job stories will be addressed by refactoring in #181
